docs/data/destinations/customerio.md

```diff
@@ -1,95 +1,70 @@
 
 1. In Amplitude Data, click **Catalog** and select the **Destinations** tab.
 2. In the Event Streaming section, click **Customer.io**.
 3. Enter a sync name, then click **Create Sync**.
 
-After you create the destination, you must configure the settings.
+### Enter credentials
+
+1. Enter your **Customer.io Tracking Site ID**.
+2. Enter your **Customer.io Tracking API Key**.
+
+### Configure mappings
+
+_This applies to both event and user forwarding. Transformed user properties aren't supported._
+
+Select an Amplitude user property that corresponds to your **Customer.io User Identifier**, from the left dropdown.
 
-### Configure settings
+- If the selected Amplitude user property values contains email addresses, Customer.io will match users on the [Customer.io email](https://customer.io/docs/identifying-people/#identifiers) (case-insensitive).
+- If the selected Amplitude user property values are prefixed with `cio_`, Customer.io will match users on the [Customer.io canonical identifier](https://customer.io/docs/identifying-people/#cio_id), a unique identifier provided by Customer.io for each user.
+- Customer.io limits the **Customer.io User Identifier** values to be no more than 150 bytes in size.
 
-1. On the **Settings** tab, click **Edit**.
-2. Enter your **Customer.io Site ID**.
-3. Enter your **Tracking API Key**.
-4. **Create & Update Users** creates users in Customer.io and update the properties of existing users when a user is created or user properties are updated in Amplitude.
-      1. To create and update users, toggle Create & Update Users to Enabled.
-      2. To select user properties to send, expand the Specify user properties to send panel, and select properties to forward. If you don't select any properties here, Amplitude doesn't include any.
-5. Configure **Send Events** to send events ingested by Amplitude to Customer.io.
-      1. To send an event, toggle **Send Events** to **Enabled**.
-      2. Expand the **Select and filter events** panel, and select which events to send.
-      3. To select event properties to send, expand the **Specify event properties to send** panel, and select properties you want to include. If you don't select any properties here, Amplitude doesn't send any.
-6. Save when finished.
+### Configure event forwarding
 
-See the [full list of available mappings](#list-of-available-mappings).
+Under **Send Events**, make sure the toggle is enabled ("Events are sent to Customer.io") if you want to stream events to Customer.io. When enabled, events are automatically forwarded to Customer.io when they're ingested in Amplitude. Events aren't sent on a schedule or on-demand using this integration. Events are sent to Customer.io as [Customer.io events](https://www.customer.io/docs/api/track/#tag/Track-Events), including web page views and mobile screen views. Customer.io automatically creates a new user in Customer.io if the provided **Customer.io User Identifier** doesn't exist in Customer.io.
```

Comment:
These were fine when they were shorter for other destinations but here they read more like bullet points and might be hard to follow.
Maybe bring the "considerations" heading back?
```markdown
Under **Send Events**, make sure the toggle is enabled ("Events are sent to Customer.io") if you want to stream events to Customer.io. When enabled, events are automatically forwarded to Customer.io when they're ingested in Amplitude.

#### Considerations:
- Events aren't sent on a schedule or on-demand using this integration. 
- Events are sent to Customer.io as [Customer.io events](https://www.customer.io/docs/api/track/#tag/Track-Events), including web page views and mobile screen views. 
- Customer.io automatically creates a new user in Customer.io if the provided **Customer.io User Identifier** doesn't exist in Customer.io.
```